components/region_cache_memory_engine/src/write_batch.rs

```diff
@@ -104,11 +104,13 @@ impl RangeCacheWriteBatch {
         Ok(())
     }
 
-    fn write_impl(&mut self, seq: u64) -> Result<()> {
+    // `seq`` is the sequence number of the first key in this write batch. Seq is
+    // incremented for each key so that all keys have unique sequence numbers.
```

Comment:
```suggestion
    // Do write coincident with Rocksdb does.
    // Attention, `seq` is the sequence number of the first key in this write batch, 
    // which will be incremented automatically for each key, so that all keys have
    // unique sequence numbers. 
```